### PR TITLE
Incorp downpayment

### LINKED ISSUE #37: Add downpayment to input

### PR DESCRIPTION
**Pull request by Mogtaba-Alim**
Added downpayment to CarBuyer entity and changed all subsequent classes that use the entity.
Made changes to the tests to cover these changes. 
Closes #37

### CODE REVIEW

File: src/components/UserForm.tsx [hunk 1]
```diff
@@ -11,12 +11,13 @@ const UserForm = () => {
   const [year, setYear] = useState(0);
   const [kilometres, setKilometres] = useState(0);
   const [price, setPrice] = useState(0);
+  const [downpayment, setDownPayment] = useState(0);
```
Comment (mtoohey31) [inline src/components/UserForm.tsx:14]: If we're going to stick to having downpayment as one word in our internal implementation, we should probably make this `setDownpayment`. I wouldn't be against using `downPayment` everywhere, that would require a lot of changes though, and the constent string for the Senso API does have to be `downpayment`, your call though.

**Review by mtoohey31 — CHANGES_REQUESTED** [review]
Looks good, just one thing with the case of the function, lmk what you decide on with that.

File: src/components/UserForm.tsx (hunk 1 above)
Comment (Mogtaba-Alim) [reply src/components/UserForm.tsx:14]: Yea, good point

**Review by mtoohey31 — APPROVED** [review]
Looks good, thanks for making that change!

**Pull request merged**